.github/workflows/ci-wheels.yml

```
@@ -0,0 +1,91 @@
+# Reference:
+#   - https://github.com/actions/checkout
+#   - https://github.com/actions/download-artifact
+#   - https://github.com/actions/upload-artifact
+#   - https://github.com/pypa/build
+#   - https://github.com/pypa/gh-action-pypi-publish
+#   - https://test.pypi.org/help/#apitoken
+
+name: ci-wheels
+
+on:
+  pull_request:
+
+  push:
+    tags:
+      - "v*"
+
+concurrency:
```

```suggestion
      - "v*"
   branches-ignore:
      - "auto-update-lockfiles"
      - "pre-commit-ci-update-config"
      - "dependabot/*"

concurrency:
```

We only need to test `main`. These other branches already get tested via `pull_request:`